survey-sparrow-android-sdk/src/main/java/com/surveysparrow/ss_android_sdk/SsSurveyFragment.java

```diff
@@ -158,20 +203,52 @@ public void onResponse(String response) {
         ssWebView.getSettings().setDomStorageEnabled(true);
         ssWebView.addJavascriptInterface(new JsObject(), "SsAndroidSdk");
 
+        ConstraintLayout constraintLayout = new ConstraintLayout(getActivity());
+        constraintLayout.setLayoutParams(new ConstraintLayout.LayoutParams(
+                ConstraintLayout.LayoutParams.MATCH_PARENT,
+                ConstraintLayout.LayoutParams.MATCH_PARENT
+        ));
+
         final ImageButton closeButton = new ImageButton(getActivity());
         closeButton.setImageResource(R.drawable.ic_close_black);
         closeButton.setBackgroundResource(R.drawable.ic_close_rounded);
         closeButton.setClickable(true);
         closeButton.setPadding(10, 10, 10, 10);
         closeButton.setVisibility(View.GONE);
 
-        FrameLayout.LayoutParams closeButtonParams = new FrameLayout.LayoutParams(
-            FrameLayout.LayoutParams.WRAP_CONTENT,
-            FrameLayout.LayoutParams.WRAP_CONTENT
+        ConstraintLayout.LayoutParams closeButtonParams = new ConstraintLayout.LayoutParams(
+                ConstraintLayout.LayoutParams.WRAP_CONTENT,
```

Comment:
@KalaiselvanK-018 ,  Any specific reason to change it from FrameLayout to ConstraintLayout ?